src/state/queries/feed.ts

```diff
@@ -299,6 +299,34 @@ export function useSearchPopularFeedsMutation() {
   })
 }
 
+const popularFeedsSearchQueryKeyRoot = 'actor-search'
```

Comment:
I think you copy-pasted the wrong key name

Reply:
Ah good catch